app/services/vendor_validator_service.rb

```diff
@@ -0,0 +1,45 @@
+class VendorValidatorService
+  TTL = Figaro.env.session_timeout_in_minutes.to_i.minutes.seconds.to_i
+
+  def submit_job(vendor_validator_class:, idv_session:, vendor_params:)
+    result_id = SecureRandom.uuid
+
+    idv_session.async_result_id = result_id
+
+    VendorValidatorJob.perform_now(
```

Comment:
Nope! I wanted to make this synchronous because then we don't have to touch the UI in this PR. Next PR will be asynchronous